manual/core/integration/README.md

```diff
@@ -0,0 +1,287 @@
+## Integration
+
+This page contains various information on how to integrate the driver in your application.
+
+### Minimal project structure
+
+We publish the driver to [Maven central][central_oss]. Most modern build tools can download the
+dependency automatically.
+
+#### Maven
+
+Create the following 4 files:
+
+```
+$ find . -type f
+./pom.xml
+./src/main/resources/application.conf
+./src/main/resources/logback.xml
+./src/main/java/Main.java
+```
+
+##### pom.xml 
+
+This is the [Project Object Model][maven_pom] that describes your application. We declare the
+dependencies, and tell Maven that we're going to use Java 8:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.example.yourcompany</groupId>
+  <artifactId>yourapp</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.datastax.oss</groupId>
+      <artifactId>java-driver-core</artifactId>
+      <version>4.0.0-beta1</version>
+    </dependency>
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+      <version>1.2.3</version>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>1.8</source>
+          <target>1.8</target>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+</project>
+```
+
+##### application.conf
+
+```
+datastax-java-driver {
+  basic.session-name = poc
+}
+```
+
+This file is not stricly necessary, but it illustrates an important point about the driver's
+[configuration](../configuration/): you override any of the driver's default options here.
+
+In this case, we just specify a custom name for our session, it will appear in the logs.
+
+##### logback.xml
+
+For this example, we choose Logback as our [logging framework](../logging/) (we added the dependency
+in `pom.xml`). This file configures it to send the driver's `INFO` logs to the console.
+
+```xml
+<configuration>
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <encoder>
+      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+    </encoder>
+  </appender>
+  <root level="ERROR">
```

Comment:
Not sure if we should encourage users to adopt `ERROR` as the root log level.

Reply:
Changed to WARN